doc/user/layouts/partials/create-source/connector/s3/details.html

```diff
@@ -1,72 +1,70 @@
 ### S3 source details
 
 The S3 source is designed to ingest a large volume of static data from [AWS's Simple Storage
-Service][aws-s3]. S3 is Amazon Web Services' cloud object store. It is important to note that an
+Service][aws-s3], Amazon Web Services' cloud object store. It is important to note that an
 object store behaves differently than a file system does, and that those differences affect how
 Materialize interacts with objects in it.
 
 Some key differences from file systems include:
 
-* Latency on individual S3 operations is much higher than even cloud-network filesystems.
-* It is not possible to seek to a point in an object, the entire object must be downloaded from
-  the beginning in order to read to a point.
-* All object operations are atomic, and it is not possible to modify just part of an S3 object,
+* Latency on individual S3 operations is much higher than on even cloud-network filesystems.
+* It is not possible to seek to a point in an object; Materialize must download the entire object
+  the beginning to read to a point.
```

Comment:
I think that the "the beginning" no longer makes sense, perhaps you didn't mean to omit the "from", so "download the entire object from the beginning to read to a point."?